src/main/java/io/jenkins/plugins/cloudevents/JSONUtil.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.cloudevents;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Converts POJO to JSON-String.
+ */
+
+public class JSONUtil {
```

Comment:
name this just CloudEventUtils for now instead of JSONUtil. You might add more utilities in the future. Even if you don't, the name won't narrow the scope of utilities which can be added here.